### DISCUSSION TITLE
pooling parameters df_error = "Inf"

### DISCUSSION THREAD

**Discussion by ndsubison2178** [amices/mice#613] [Impute, analyse and pool]
What is the reason for seeing values like "Inf" in column df_error ? when using pool_parameter function to pool results after fitting model on mice imputed dataset ? Thanks.

**Comment by hanneoberman** [comment]
Could you provide a [`reprex`](https://reprex.tidyverse.org/) for this behavior? I'm suspecting it is not an issue in the software, so your question can be answered on our Discussion forum.
